ci-operator/step-registry/openshift/e2e/openstack/ipi-e2e-only/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
Replace this symlink with an actual OWNERS file containing your approvers and your reviewers